frontend/web/components/modals/payment/pricingFeatures.tsx

```diff
@@ -0,0 +1,75 @@
+import { PricingFeature } from './types'
```

Comment:
NIT: Can we maybe rename this file to add config or constants for later brain satisfaction?